TTS/tts/utils/text/__init__.py

```diff
@@ -6,6 +6,7 @@
 
 from TTS.tts.utils.text import cleaners
 from TTS.tts.utils.text.chinese_mandarin.phonemizer import chinese_text_to_phonemes
+from TTS.tts.utils.japanese import japanese_text2phone
```

Comment:
Can you move your code under `TTS.tts.utils.text` and rename to `japanese_text_to_phonemes` to align with the Chinese API ?

Reply:
Updated.  Just in case, 
- `phoneme_language` is all lower cased except `zh-CN`. Not sure which I should align with.
- Loading MeCab package is a bit heavy and probably Chinese is as well. We may want to make `text2phone()` as a variable so that we can lazy-load packages. We may also make those packages `extras_require` to avoid depending on unused packages for most users.

Let me try writing recipe. Thanks